truststore_java.go

```diff
@@ -29,14 +30,19 @@ var (
 )
 
 func init() {
+	keytoolPath = "bin/keytool"
+	if runtime.GOOS == "windows" {
+		keytoolPath += ".exe"
+	}
+
 	if v := os.Getenv("JAVA_HOME"); v != "" {
 		hasJava = true
 		javaHome = v
 
-		_, err := os.Stat(path.Join(v, "bin/keytool"))
+		_, err := os.Stat(path.Join(v, keytoolPath))
 		if err == nil {
 			hasKeytool = true
-			keytoolPath = path.Join(v, "bin/keytool")
+			keytoolPath = path.Join(v, keytoolPath)
 		}
 
 		cacertsPath = path.Join(v, "jre/lib/security/cacerts")
```

Comment:
Is this path created by the java installer on macos? I ran a fresh install of jdk 10 on win10 and don't see this path anywhere. I can try it on my mac as well, but I would imagine the path would be at the very least something like `/jre10.0.1/` or something like that.

Reply:
This was originally written on a mac yea. If you find the windows path is diffetent let me know and I'll fix!

Reply:
I just installed the jdk 10 and I don't see that path either. I'm looking at `/Library/Java/JavaVirtualMachines/` which appears to be similar to windows, but still no jre folder.

Reply:
`/Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/lib/security/cacerts` is the path for jre-10.0.2 on my mac also. Are you installing the server jre on your machine?